optaplanner-examples/src/main/java/org/optaplanner/examples/common/app/CommonApp.java

```diff
@@ -151,9 +151,12 @@ protected AbstractSolutionExporter createSolutionExporter() {
         return null;
     }
 
-    public Solution_ getUnsolvedProblem(String problemFileName) {
+    public Solution_ loadUnsolvedProblemFromResource(String problemFileName) {
         String pathToResource = String.format("/org/optaplanner/examples/%s/data/%s", dataDirName, problemFileName);
         String fileName = getClass().getResource(pathToResource).getFile();
         return createSolutionFileIO().read(new File(fileName));
     }
+    public Solution_ loadUnsolvedProblemFromDataFolder(String problemFileName) {
+        return createSolutionFileIO().read(new File(determineDataDir(dataDirName),"unsolved"+ File.separator + problemFileName));
```

Comment:
Missing spaces around "unsolved". Would be nice if the line fits into 120 characters.